sdk/src/androidTest/java/com/mapbox/maps/MapboxMapIntegrationTest.kt

```diff
@@ -34,7 +34,11 @@ class MapboxMapIntegrationTest {
   @UiThreadTest
   @Test
   fun testGetResourceOptions() {
-    val defaultOptions = MapboxOptions.getDefaultResourceOptions(mapView.context)
+    val defaultOptions = MapboxMapOptions(
+      mapView.context,
+      mapView.context.resources.displayMetrics.density,
+      null
```

Comment:
Perhaps I'm crazy but how can null be passed here when we have non-nullable parameter of `CredentialsManager` in `MapboxMapOptions` ctor?

Reply:
It's for `val attrs: AttributeSet? = null`

Reply:
it's null by default, can we remove this `null` then?